### ISSUE TITLE
Total Number of Torrents in Active State - Settings

### ISSUE THREAD

**Issue by p4cm4n88** [johman10/flood-for-transmission#123]
This is more of a feature request, but maybe I have missed this far?
In settings, unless I'm missing it - there is not an option for 'total number of active torrents', so for example if I typically have 20 active torrents, but would like to bump that up, I don't see that option.

**Comment by johman10** [comment]
@p4cm4n88 you're correct. This configuration is currently missing from the UI, and there are a couple more.

I documented them here: https://github.com/johman10/flood-for-transmission/blob/master/src/components/Modal/Settings/Settings.svelte#L14-L26

I will have a look at implementing this, but there are two other things before I will get to this. So it might take a while.

Thanks for the feedback, I will make this a priority because of it. ;)

**Comment by johman10** [comment]
As you can see I just merged some code to make this work. It also included some other configuration options, I hope it's of value and let me know if something doesn't work as expected or if something isn't clear! :)
